temba/msgs/models.py

```diff
@@ -1212,7 +1206,7 @@ def create_incoming(cls, channel, urn, text, user=None, date=None, org=None, con
             analytics.gauge('temba.msg_incoming_%s' % channel.channel_type.lower())
 
         # ivr messages are handled in handle_call
-        if status in (PENDING, INTERRUPTED) and msg_type != IVR:
+        if status == PENDING and msg_type != IVR:
```

Comment:
Won't this match USSD case? (guessing we are treating IVR and USSD the same in msg handling)